dialog_manager/FrameMessage[] frames
